core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.ts

```diff
@@ -114,11 +117,11 @@ angular.module('oppia').component('settingsTab', {
         ExplorationParamSpecsService, ExplorationRightsService,
         ExplorationStatesService, ExplorationTagsService,
         ExplorationTitleService, ExplorationWarningsService,
-        UrlInterpolationService, UserEmailPreferencesService,
+        RouterService, UrlInterpolationService, UserEmailPreferencesService,
         UserExplorationPermissionsService, WindowRef, ALL_CATEGORIES,
         EXPLORATION_TITLE_INPUT_FOCUS_LABEL, TAG_REGEX) {
       var ctrl = this;
-
+      ctrl.directiveSubscriptions = new Subscription();
```

Comment:
Move this a few lines down below the constants. Try not to break up logical groupings of code.

Reply:
Done